app/components/organisms/HeaderWithSearch.tsx

```diff
@@ -8,7 +8,7 @@ export default function HeaderWithSearch() {
 	const searchBar = isOnSearchPage ? null : <SearchBar status="idle" />
 
 	return (
-		<header className="bg-primary/10 py-6 dark:bg-dark-primary/10">
+		<header className="dark:bg-dark-primary/10 bg-primary/10 py-6">
 			<nav className="container flex flex-wrap items-center justify-between gap-4 sm:flex-nowrap md:gap-8">
 				<div className="ml-auto hidden max-w-sm flex-1 sm:block">
```

Comment:
See if you can add a logo into the nav bar, as well as your footer.

See the [guide](https://gedsargent.github.io/code-monster/guides/epic-news/ui/import-image-to-navbar-and-footer/#importing-the-image) on Code Monster to get help with this..